idd/Energy+.idd.in

```diff
@@ -40246,15 +40250,17 @@ ZoneHVAC:TerminalUnit:VariableRefrigerantFlow,
         \note This field is set to zero flow when the VRF terminal unit is connected to
         \note central dedicated outdoor air through air terminal single duct mixer object.
   A5 ,  \field Supply Air Fan Operating Mode Schedule Name
-        \required-field
         \type object-list
         \object-list ScheduleNames
+        \note Required for zone equipment. Leave blank if terminal unit is used in AirLoopHVAC:OutdoorAirSystem:EquipmentList.
+        \note Also leave blank if terminal unit is used on main AirloopHVAC branch and terminal unit has no fan.
   A6 ,  \field Supply Air Fan Placement
         \type choice
         \key BlowThrough
         \key DrawThrough
-        \default BlowThrough
```

Comment:
Deleting the default could break existing idfs that have this field blank, expecting it to fill with BlowThrough.  You could just say it's ignored if the terminal unit has no fan?